src/main/java/com/stripe/exception/ApiKeyMissingException.java

```diff
@@ -0,0 +1,20 @@
+package com.stripe.exception;
+
+/**
+ * {@link ApiKeyMissingException} is thrown when the API key is not set for a request. The API key
+ * for a request may be set either globally through {@link com.stripe.Stripe#apiKey} or through
+ * {@link com.stripe.net.RequestOptions}.
+ */
+public class ApiKeyMissingException extends RuntimeException {
+  private static final long serialVersionUID = 1L;
+
+  /** Initializes a new instance of the {@link ApiKeyMissingException}. */
```

Comment:
Do we need this constructor? Maybe we should always pass a message explaining the fix?

Reply:
Do you mean in the constructer, do something like `super('API key missing')`, or do you mean when we throw the exception?

Reply:
We shouldn't provide an empty constructor, and the call sites which new up an exception should pass an error message explaining the issue + fix.